media/js/base/mozilla-fxa-form.js

```diff
@@ -49,21 +48,39 @@ if (typeof window.Mozilla === 'undefined') {
 
             fxaSubmitButton.disabled = false;
 
-            var destURL = fxaForm.getAttribute('action') + 'metrics-flow';
+            // track external UTM referrals
+            var urlParams = new window._SearchParams().utmParamsFxA();
+            var attributionData = Mozilla.UtmUrl.getAttributionData(urlParams);
+
+            if (attributionData) {
+                var utms = ['utm_source', 'utm_campaign', 'utm_content', 'utm_term', 'utm_medium'];
+                for (var i = 0; i < utms.length; i++) {
+                    if (attributionData.hasOwnProperty(utms[i])) {
+                        fxaForm.querySelector(`[name="${utms[i]}"]`).value = `${attributionData[utms[i]]}`;
```

Comment:
tests are failing due to the notation on this line: 
```
(`[name="${utms[i]}"]`).value = `${attributionData[utms[i]]}`
```
but I'm not quite sure how to achieve this without the back-ticks?

Reply:
String template literals don't yet have broad enough browser support to use natively: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

You'll probably need to use regular string concatenation instead:

```javascript
'[name="' + utms[i] + '"]'.value = attributionData[utms[i]]
```